cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -106,22 +107,49 @@ def internal_notes(self, key, value):
     return _internal_notes
 
 
-# TODO convert to tags
-@model.over("_migration", "(^980__)|(^690C_)|(^697C_)")
+@model.over("tags", "(^980__)|(^690C_)|(^697C_)")
 @out_strip
-def collection(self, key, value):
-    """Translates collection field - WARNING - also document type field."""
-    _migration = self["_migration"]
-    _tags = _migration["tags"]
+def tags(self, key, value):
+    """Translates tag field - WARNING - also document type and serial field."""
+    _tags = self.get("tags", [])
     for v in force_list(value):
         result_a = mapping(COLLECTION, clean_val("a", v, str))
         result_b = mapping(COLLECTION, clean_val("b", v, str))
         if result_a:
             _tags.append(result_a) if result_a not in _tags else None
-            _migration["has_tags"] = True
         if result_b:
             _tags.append(result_b) if result_b not in _tags else None
-            _migration["has_tags"] = True
+        if not result_a and not result_b:
+            serial(self, key, value)
+    return _tags
+
+
+@model.over("_migration", "(^980__)|(^690C_)|(^697C_)")
+def serial(self, key, value):
```

Comment:
I think it could be more clear if this function name is `special_serials` so it is specific to what is happening in the mapping